include/freerdp/server/shadow.h

```diff
@@ -204,7 +204,7 @@ typedef struct _SHADOW_MSG_OUT SHADOW_MSG_OUT;
 typedef void (*MSG_OUT_FREE_FN)(UINT32 id, SHADOW_MSG_OUT* msg);
 #define RDP_SHADOW_MSG_OUT_COMMON() \
 	int refCount; \
-	MSG_OUT_FREE_FN Free; /* function to free SHADOW_MSG_OUT */
+	MSG_OUT_FREE_FN Free /* function to free SHADOW_MSG_OUT */
```

Comment:
@akallabeth @bmiklautz 
I'll delete the comments :) And put the comment in MSG_OUT_FREE_FN instead. I think it's clear enough.

It doesn't break compiler on win and Linux, but I agree it looks risky.